### DISCUSSION TITLE
Decreased charge rate, exporting to grid instead of charging, POD_w045_SW_Brick_OV_Warning

### DISCUSSION THREAD

**Discussion by BJReplay** [jasonacox/Powerwall-Dashboard#380] [Q&A]
I suspect I might have a failing PW.

It is set to self consumption.

Happening right now...

![image](https://github.com/jasonacox/Powerwall-Dashboard/assets/37993507/9e04d471-6fe4-492b-9d68-38f15fdc813f)

Over the last few days, as it gets to 90% charge, it starts exporting, and tailing off the charge rate.  Once it gets to reported 100% charge, it then starts exporting from the powerwall at around 300W until the insolation drops down, and the SOC drops below reported 100%.

Here's a second shot from yesterday where the tapering charge rate can be clearly seen.

Tapering charge rate circled (badly drawn) in red.

Post 100% SOC discharge circled in pink.

Decreased pack charge circled in blue.

![image](https://github.com/jasonacox/Powerwall-Dashboard/assets/37993507/84ecee44-94aa-42b8-a555-935581d413f8)

About to call Tesla and see what they say.

**Comment by jasonacox** [comment]
Fascinating, @BJReplay - any success? Also, out of curiosity, with the OV warning, did you see anything odd in the voltage data?

**Comment by BJReplay** [comment]
> any success

No, not really.  They said it's normal, and when I said, hang on, look at this day, it's quite different, they said they'd monitor it, but there are no faults.

I asked them to look at this day from about a week ago (October 20)

![image](https://github.com/jasonacox/Powerwall-Dashboard/assets/37993507/73fd8267-bab0-40a0-b153-8a6ec9433a4e)

And compare it to this day (yesterday - it got to 100% and then started discharging while I was on the phone) where the transition from charging to full is quite different, and it immediately starts discharging the battery once full, but they're saying "nothing to see".

![image](https://github.com/jasonacox/Powerwall-Dashboard/assets/37993507/7775e92c-5991-45b7-94e2-5f55d6b8af58)

> did you see anything odd in the voltage data?

No, not around the time.

I'm assuming the OV is OverVoltage and UV in other alerts is UnderVoltage.

**Reply by jasonacox** [reply]
> I'm assuming the OV is OverVoltage 

Yes, that is what I was thinking too. I would suspect that the battery would trip on over voltage (something outside the band of acceptable for charging), but it could be that the alert just happens to be a side effect of whatever is happening with your batteries.

> they're saying "nothing to see".

Ugh... Maybe it is one of those things where it isn't worth their effort to troubleshoot and to wait for it to to fully fail. But that drives me nuts. I have to start to wonder if they are not loving those of us with monitoring tools / dashboards... :-)

**Comment by BJReplay** [comment]
> I have to start to wonder if they are not loving those of us with monitoring tools / dashboards

For sure!  That said, I didn't disclose that I was using a tool to monitor; I just said that if you look at the app in the combined view, when you click on the eye icon, you can see it really clearly 😉 and the CSA did say "I'm looking in the app and I can see what you mean".
